docs/tutorials/image_classification.md

```diff
@@ -147,6 +147,7 @@ system:
 
     optimizer:
         _target_: torch.optim.Adam
+        params: "$@system#model.parameters()" # Link to model's learnable parameters
```

Comment:
Copied from other parts of the docs. `params` is mandatory, so this was necessary.